app/test_main.py

```diff
@@ -1,3 +1,44 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, human_years",
+    [
+        pytest.param(
+            0, 0, [0, 0],
+            id="cat/dog years = 0"
+        ),
+        pytest.param(
+            14, 14, [0, 0],
```

Comment:
The test case with `cat_age` and `dog_age` both set to 14 and expecting `[0, 0]` might not be correct if the logic for converting these ages to human years is supposed to start counting from 15. Please verify if this is the intended behavior.